curriculum/challenges/english/16-the-odin-project/top-the-box-model/the-box-model-question-b.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 64a553ed02815a7d323aab89
+title: The Box Model Question B
+challengeType: 15
+dashedName: the-box-model-question-b
+---
+# --description--
+
+In the Elements pane, you can see the entire HTML structure of your page. You can click on any of the elements in this pane to select that specific element. Alternatively, you can click the blue-highlighted icon shown below on the left, and hover over any element on the page.
+
+![Inspector Icon](https://cdn.statically.io/gh/TheOdinProject/curriculum/594984d7c9f9e744577f19ea475b3864e8cc7c91/html_css/v2/foundations/inspecting-html-and-css/imgs/01.png)
```

Comment:
I wonder if we want to put these images on our own CDN.

Reply:
Yes